tests/test_app.py

```diff
@@ -626,10 +626,12 @@ async def test_list():
 
 
 # remove deadline from hypothesis, since this test can be laggy and lead to false-negative fails
-@settings(deadline=None)
-@asyncgiven(txt=st.text())
-async def test_download(txt):
-    async with await AppClient.create(CONFIG, TEST_TOKEN) as cli:
+# ignore hypothesis warning about function scoped fixture
```

Comment:
Some notes on this:
I set the `config_temp_dir` fixture to be function scope, so it wipes out the test DTS staging data dir on each run (will be used later, hopefully in the next commit or two, for tests).

Hypothesis wraps tests to run multiple times in the same scope, so that directory doesn't get wiped (and all function-scoped fixtures don't return until all Hypothesis trials are run). This setting just keeps hypothesis from throwing an error in this case.